src/main.rs

```diff
@@ -1,7 +1,3 @@
-#[macro_use]
-extern crate gb_io;
-extern crate regex;
```

Comment:
With the 2018 edition the `extern crate` and `#[macro_use]` are not needed anymore in most cases, and you can specify with `use` directly.